tests/test_data/test_pipelines/test_indoor_sample.py

```diff
@@ -1,14 +1,13 @@
 import numpy as np
 
 from mmdet3d.core.points import DepthPoints
-from mmdet3d.datasets.pipelines import (IndoorPatchPointSample,
-                                        IndoorPointSample,
+from mmdet3d.datasets.pipelines import (IndoorPatchPointSample, PointSample,
                                         PointSegClassMapping)
 
 
 def test_indoor_sample():
```

Comment:
Maybe we should add a case to test `PointSample` with `sample_range is not None`? After all, this is a newly introduced function.

Reply:
Thanks for your advice. However, there needs a data conversion from LiDAR to CAMERA or DEPTH before PointSample, which is not supported by the current repo. After a future merge of function 'data conversion', I will add an additional unittest.

Reply:
But I guess in unit tests, we can manually construct some Camera points and use them as input? I don't think it's a good idea to 'tie' two transforms together and only test them together? Separately testing makes fewer bugs I believe.

Reply:
I just added an unittest in `test_transforms_3d.py` to test the point sampling function.

Reply:
Yes, great!